<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>segunda</class>
 <widget class="QMainWindow" name="segunda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.532, y2:1, stop:0.174129 rgba(17, 37, 232, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Crear Cuenta</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_crear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Correo Electronico</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrasenia_crear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nombre_crear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Nombre Completo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Puesto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>PUESTO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gerente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Empleado</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="Codigo_Gerente">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Codigo Gerente (Si es Empleado Codigo: 1234)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Create_Account_Button_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 87, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
